unittests/AST/ASTImporterTest.cpp

```diff
@@ -814,6 +814,56 @@ TEST(ImportDecl, ImportFunctionTemplateDecl) {
               has(returnStmt(has(integerLiteral(equals(2)))))))))))))))))));
 }
 
+TEST(ImportDecl, DISABLED_ImportTemplateDefaultArgument) {
+  MatchVerifier<Decl> Verifier;
+  EXPECT_TRUE(
+        testImport(
+          "template <typename T=int> void declToImport(T &t) { };",
+          Lang_CXX11, "", Lang_CXX11, Verifier,
+          functionTemplateDecl(has(templateArgument()))));
+}
+
+TEST_F(Fixture,
+       ImportTemplatedDecl) {
+  auto Code =
+        R"(
```

Comment:
This code could be a one-line string literal rather than a multi-line raw string literal